www/docs/en/dev/config_ref/index.md

```diff
@@ -58,7 +58,8 @@ platform. See [Icons and Splash Screens](images.html) for more information.
    id(string) | *Required* <br/> Specifies the app's reverse-domain identifier, and the `version` its full version number expressed in major/minor/patch notation.
    version(string) | *Required* <br/> Full version number expressed in major/minor/patch notation.
    versionCode(string) | ==Android== <br/> Alternative version for Android. For further details, see [Android versioning](http://developer.android.com/tools/publishing/versioning.html)
-   CFBundleVersion(string) |  ==iOS== <br/> Alternative version for iOS. For further details, see [iOS versioning](https://developer.apple.com/library/ios/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102364).
+   ios-CFBundleVersion(string) |  ==iOS== <br/> Alternative version for iOS. For further details, see [iOS versioning](https://developer.apple.com/library/ios/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102364).
+   osx-CFBundleVersion(string) |  ==OS X== <br/> Alternative version for OS X. For further details, see [OS X versioning](https://developer.apple.com/library/prerelease/mac/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102364).
```

Comment:
no, it's not hard. but we should discuss this on the list. `ios-CFBundleVersion` and `ios-CFBundleIdentifier` are the current used ones:
- https://github.com/apache/cordova-lib/blob/6.0.0/cordova-common/src/ConfigParser/ConfigParser.js#L106
- https://github.com/apache/cordova-lib/blob/6.0.0/cordova-common/src/ConfigParser/ConfigParser.js#L132